packages/api/src/schema/api-schema.yaml

```
@@ -2223,59 +2280,76 @@ components:
         device:
           type: string
           description: The device used by the viewer.
+          example: iPhone
         deviceType:
           type: string
           description: The type of the device used by the viewer.
+          example: mobile
         cpu:
           type: string
           description: The CPU used by the viewer's device.
+          example: ARM
         os:
           type: string
           description: The operating system used by the viewer.
+          example: iOS
         browser:
           type: string
           description: The browser used by the viewer.
+          example: Safari
         browserEngine:
           type: string
           description: The browser engine used by the viewer's browser.
+          example: WebKit
         continent:
           type: string
           description: The continent where the viewer is located.
+          example: North America
         country:
           type: string
           description: The country where the viewer is located.
+          example: United States
         subdivision:
           type: string
           description: |
             The subdivision (e.g., state or province) where the viewer is
             located.
+          example: California
         timezone:
           type: string
           description: The timezone where the viewer is located.
+          example: America/Los_Angeles
         geohas:
```

Is this actually like this? Sounds like maybe a typo
```suggestion
        geohash:
```